### PR TITLE
Dpl-924 scRNA - additional scRNA config to allow creation of Dil plate

### PR DESCRIPTION
**Pull request by andrewsparkes**
#### Changes proposed in this pull request
Additional WIP-flagged configuration for the scRNA pipeline around the aggregation and start of the 5p library prep section of the pipeline.

### CODE REVIEW

File: config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml [hunk 1]
```diff
@@ -35,3 +35,25 @@ limber_scrna_core_cdna_prep_input:
     # - Chromium single cell 3 prime HT v3
     # - Chromium single cell BCR HT
     # - Chromium single cell TCR HT
+limber_scrna_core_chromium_ht_aggregation:
```
Comment (KatyTaylor) [inline config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml:38]: Request type and submission template names were planned to be different here - https://github.com/sanger/limber/issues/1444 - see which you prefer

Comment (yoldas) [reply config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml:38]: @KatyTaylor I thought these request types are scRNA Core specific. The key names are not visible in the UI.
The submission templates names also include scRNA Core and these request type keys listed under them. 

For submission template names, including scRNA Core in the name allows user to find them by typing in the dropdown-search https://uat.sequencescape.psd.sanger.ac.uk/submissions/new .

Comment (andrewsparkes) [reply config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml:38]: I saw the planned ones and didn't like how vague they were. We're copying the aggregation and chromium pipelines for specific use in scRNA so the names should contain scRNA I think.
Or am I misunderstanding and these latter parts of the pipeline are to be used by non-scRNA stuff as well?
It's all very well wanting to build generic re-useable pipelines but it never turns out that way, there is always pipeline specific configuration, or files, or binning rules, or something that diverges.

Comment (KatyTaylor) [reply config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml]: OK. I'm not sure of the pros and cons of generic vs specific request type naming. Like Abdullah says, they're not things seen by the users. Making specific ones gives us more data to maintain, and more constraints. What are the pros of specific naming?

**Review by KatyTaylor — APPROVED** [review]

File: config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml (hunk 1 above)
Comment (yoldas) [reply config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml]: I think, it makes development easier, prevents side effects and allows customisation.

**Pull request merged**